src/Uno.UI.WpfHost/Uno.UI.WpfHost.csproj

```diff
@@ -93,7 +93,7 @@
       <Version>67.0.0</Version>
     </PackageReference>
     <PackageReference Include="CefSharp.Wpf">
-      <Version>67.0.0</Version>
```

Comment:
This will mean consumers of the `Uno.UI.WpfHost` nuget package will need to use at min v73.1.130 of CEF. I think always forcing consumers to upgrade to the latest of CEF is merited as CEF is web-browser and updates will typically include security fixes. Thoughts @unoplatform/maintainers?